storybook/qmlTests/tests/tst_SwapModal.qml

```diff
@@ -189,6 +197,7 @@ Item {
         }
 
         function test_floating_header_list_items() {
+            skip("Randomly failing")
```

Comment:
@Khushboo-dev-cpp I'll skip 2 tests here because it's randomly failing on master. Do you want me to add a task for them, or should I mention them in an existing one?

Reply:
a new task would be great, I'll work on it first thing next sprint!